src/H5Ipkg.h

```diff
@@ -371,7 +379,8 @@ typedef struct H5I_suint64_t {
  *
  * num_id_info_fl_alloc_req_denied_due_to_no_reallocable_entries:  Number of times an 
  *      alloc request for an instance of H5I_mt_id_info_t has had to be satisfied from
- *      the heap because num_id_info_fl_entries_reallocable is zero.
+ *      the heap because the serial number of the entry at the head of the free list
+ *      has serial number greater than or equal to id_max_realloc_sn,
```

Comment:
`the serial number of the entry at the head of the free list has` -> `the entry at the head of the free list has``